Makefile

```diff
@@ -1,85 +1,69 @@
 SHELL := /bin/bash
 
 NAME ?= terrahelp
+BIN := $(CURDIR)/bin
 
-BUILDARGS ?= -mod=vendor
+go_files := $(shell find . -path '*/testdata' -prune -o -type f -name '*.go' -not -path "./vendor/*" -print)
 
-REQ_GO_VERSION := 1.16
-GO_VERSION := $(shell go version | sed -E 's/^go version go([0-9]+.[0-9]+.[0-9]+).*$$/\1/')
-MAX_GO_VERSION := $(shell printf "%s\n%s" $(REQ_GO_VERSION) $(GO_VERSION) | sort -V -r | head -1)
+.DEFAULT_GOAL := all
+.PHONY := all vet fmt fmtcheck test install uninstall clean dependencies vendor-dependencies tidy-dependencies clean-dependencies
 
-BIN := $(CURDIR)/bin
-DIST := $(CURDIR)/dist
-OUPUT_FILES := $(BIN) $(DIST)
+vet: $(go_files)
+	go vet  ./...
 
-PLATFORMS ?= darwin linux
-ARCH ?= amd64
-OS = $(word 1, $@)
+fmt:
+	@go run golang.org/x/tools/cmd/goimports -w $(go_files)
```

Comment:
Is it worth running the actual `go fmt` as well?

Reply:
Good question :)  `goimports` also executes a format in the style of gofmt.  So it covers the import statements and overall code format.